scout-hellojs-app/src/main/resources/archetype-resources/__rootArtifactId__.app/src/main/js/__simpleArtifactName__.ts

```diff
@@ -1,7 +1,9 @@
 import {App} from '@${simpleArtifactName}/ui';
+
 new App().init({
   bootstrap: {
-    textsUrl: 'texts.json'
+    textsUrl: 'texts.json',
+    permissionsUrl: 'api/permissions'
```

Comment:
convention over configuration? allow to customize URL but use a default (e.g. hardcoded resource url) if no specific URL is given?